src/arizona_live_view.erl

```diff
@@ -26,49 +26,43 @@ Live view.
 %% API functions.
 -export([parse_str/2]).
 -ignore_xref([parse_str/2]).
--export([compile/3]).
--ignore_xref([compile/3]).
--export([persist_get/3]).
```

Comment:
This was basically just

```erlang
{ok, Compiled} = compile(Mod, Fun, Macros)
```

so was removed.

`compile/3`, otoh, is basically also just

```erlang
arizona_tpl_compile:compile({Mod, Fun, Macros})
```

so was removed (it's not wrapping much!).